spring-integration-jmx/src/test/java/org/springframework/integration/jmx/UpdateMappingsTests.java

```diff
@@ -95,7 +95,7 @@ public void testJmx() throws Exception {
 		MessagingTemplate messagingTemplate = new MessagingTemplate();
 		messagingTemplate.setReceiveTimeout(1000);
 		Set<ObjectName> names = this.server.queryNames(ObjectName
-				.getInstance("update.mapping.domain:type=HeaderValueRouter,name=router"),
+				.getInstance("update.mapping.domain:type=MessageHandler,name=router,bean=endpoint"),
```

Comment:
Previously, two MBeans were registered for the router. Now there's only one (with all operations).